tests/testing_zpotrf.c

```diff
@@ -18,7 +18,8 @@ int main(int argc, char ** argv)
 {
     parsec_context_t* parsec;
     int iparam[IPARAM_SIZEOF];
-    dplasma_enum_t uplo = dplasmaUpper;
```

Comment:
Don't forget to undo that at the end when the U is also done

Reply:
OK

Reply:
Why do you map the A matrix on lower ? You don't need to alter this variable, instead just call the function with `dplasmaLower` and add a comment on why you choose to do so.

Reply:
`uplo` is used throughout the testing; I will create an issue to track the performance difference between lower and upper as shown in the table in the PR.